### ISSUE TITLE
[MAS4.1.2] Name of Audio clip section is not properly defined.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1313]
**Actual Result:**
Name of Audio Clip section is not properly defined. So, Screen reader is reading 'Audio' twice for Audio clip present on the page.

**Expected Result:**
Name of Audio Clip section should be properly defined. So, Screen reader should read 'Audio' once only with proper description for Audio clip present on the page.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/ ) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'accessibility' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Page will upload.
5. Open Narrator (Ctrl+Win. key+Enter) 
6. Navigate to audio clip by using Tab key.
7. Observe that narrator reads audio twice when focus lands on the audio clip present on the page.

**User Impact:**
Screen reader users may get confuse if same content is being read more than once.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Usable_reading same value 2 times.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547633/Usable_reading.same.value.2.times.zip)

**Comment by priysn** [comment]
Closing this bug as part of internal Bugs review.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
